src/main/java/org/fcrepo/spec/testsuite/crud/ExternalBinaryContent.java

```diff
@@ -349,6 +762,12 @@ public void binaryContentGuaranteeHeadersDescribedBy(final String uri) {
                         "the request entity must reject request if it cannot guarantee all of the response headers " +
                         "required by the LDP-NR interaction model in this specification.",
                 "https://fcrepo.github.io/fcrepo-specification/#external-content", ps);
+
+        final String location = createExternalBinary(uri, info.getId(), "proxy", null);
+
+        final Response resp = doGet(location);
+        assertTrue("Response does not contain link of rel type = describedby",
```

Comment:
We likely want to add more expected headers, but that can be as a follow-on ticket.